pkg/server/config.go

```diff
@@ -160,5 +169,19 @@ func (cfg *Config) ToFlags(set *cli.FlagSet) *cli.FlagSet {
 		Usage:  `The number of minutes to cache configuration files before retrieving fresh ones. Defaults to 15 minutes.`,
 	})
 
+	f.StringVar(&cli.StringVar{
```

Comment:
Use the time.Duration instead (that way you're not limited to seconds)

Reply:
Done